blog/2023-05-18-how-to-use-conda-build.mdx

```diff
@@ -12,68 +12,68 @@ import condabuild from '@site/static/img/blog/2023-05-18-how-to-use-conda-build.
 
 <ol>
 <li>A script that contains metadata.</li>
 <li>The script that installs the files for the package on macOS and Linux.</li>
 <li>The build script that installs the files for the package on Windows.</li>
-<li>Any additional setup files depending on the complexity of the package.</li>
+<li>Any additional setup files, depending on the complexity of the package.</li>
 </ol>
 
-In this article, we will guide you through the process of building your python package with conda-build! Conda-build can help us handle even the most complex dependencies for our package, making the process smoother and more efficient. Let's get ready to build an amazing Python package!
+In this article, we will guide you through the process of building your Python package with conda-build! Conda-build can help us handle even the most complex dependencies for our package, making the process smoother and more efficient. Let's get ready to build an amazing Python package!
 
 ## Getting Started 
 
 ### Prerequisites
-Let's start building our own Python package. First, you need to ensure that you have installed <a href="https://docs.anaconda.com/anaconda/install/">Miniconda or Anaconda</a>. Conda-build is a package manager that can be installed using either of the two package managers.
+Let's start building our own Python package. First, you need to ensure that you have installed <a href="https://docs.conda.io/en/latest/miniconda.html#">Miniconda</a> or <a href="https://docs.anaconda.com/anaconda/install/">Anaconda</a>. Conda-build is a package manager that can be installed using either of the two package managers.
 
 
 
 ### Installing conda-build
- Create a new [conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) and run the following command on your terminal :
+ Create a new [conda environment](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html) and run the following command on your terminal:
 ```bash
 conda install conda-build
 ```
 
 ### Creating a template for the recipe
-For this tutorial, we'll be walking you through the process of building a recipe for PyPi's "click" package. Pypi is among the various repositories available for python softwares on the internet. 
+For this tutorial, we'll be walking you through the process of building a recipe for PyPI's [`click`](https://github.com/pallets/click) package using Grayskull. Grayskull is an automatic Conda recipe generator that is used for creating recipes for Python packages on PyPI and non-PyPI packages available on GitHub. PypI is among the various repositories available for Python softwares on the internet.
```

Comment:
```suggestion
For this tutorial, we'll be walking you through the process of building a recipe for PyPI's [`click`](https://github.com/pallets/click) package using Grayskull. Grayskull is an automatic Conda recipe generator that is used for creating recipes for Python packages on PyPI and non-PyPI packages available on GitHub. PyPI is among the various repositories available for Python softwares on the internet.
```

Reply:
I have corrected this typo. I  reviewed the file and found no additional typos. Thankyou for your time and valuable suggestions!